packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -0,0 +1,71 @@
+/* stylelint-disable  */
+
+@import '~terra-mixins';
+@import './variables';
+@import './mixins';
+
+%terra-demographics-banner {
+  background: $demographics-background;
+  color: #fff;
+  display: -ms-flex;
+  display: flex;
+  margin-bottom: 10px;
+  padding: 10px;
+
+  h1 {
+    margin: 0;
+
+    .terra-DemographicsBanner--preferred-first-name {
+      margin: auto 5px;
+
+      &::before,
+      &::after {
+        content: '"';
+      }
+    }
+  }
+}
+
+.terra-DemographicsBanner {
+  @extend %terra-demographics-banner;
+}
+
+.terra-DemographicsBanner--deceased {
+  @extend %terra-demographics-banner;
```

Comment:
Rather than extension of styles via Sass extend, prefer extension of styles through a base class and additional modifier classes which can be composed in the html. This is to help reduce total file size of CSS.

e.g.
```html
<!-- Base Demographics Banner -->
<div class="terra-DemographicsBanner">...</div>

<!-- Deceased Demographics Banner -->
<div class="terra-DemographicsBanner terra-DemographicsBanner--deceased">...</div>
```

```css
.terra-DemographicsBanner {
  base styles ......
}

.terra-DemographicsBanner--deceased {
  background-color: $demographics-background-deceased;
}
```